modules/server_new/src/main/scala/observe/server/ObserveEngine.scala

```diff
@@ -1853,6 +1860,7 @@ object ObserveEngine {
                 SingleActionOp.Error.apply(_, _, _, r.msg)
               ) :: SequenceUpdated(svs) :: Nil
             )
+          case SystemEvent.AtomCompleted(_)                                         => Stream.empty
```

Comment:
I imagine the UI will mant to get an event when an atom completes. It can be done in another PR though